crates/bevy_math/src/primitives/dim2.rs

```diff
@@ -119,6 +157,13 @@ pub struct BoxedPolyline2d {
 }
 impl Primitive2d for BoxedPolyline2d {}
 
+impl BoxedPolyline2d {
+    /// Create a new `Polyline2d` from an array of vertices
+    pub fn new(vertices: Box<[Vec2]>) -> Self {
```

Comment:
We should remove this or swap it over to another iterator method.

Reply:
I swapped it to another iterator method using the `Vec` struct. Can you review it?